docs/references/javascript/sign-in/authenticate-with.mdx

```diff
@@ -66,6 +66,20 @@ function authenticateWithCoinbaseWallet(): Promise<SignInResource>
 | - | - |
 | `Promise<SignInResource>` | A `Promise` which resolves to the current [`SignIn`][signin-ref]. |
 
+### `authenticateWithOKXWallet()`
+
+Starts a sign-in flow that uses the OKX Wallet to authenticate the user using their Web3 wallet address.
+
+```typescript
+function authenticateWithOKXWallet(): Promise<SignInResource>
+```
+
+#### Returns
+
+| Type | Description |
+| - | - |
+| `Promise<SignInResource>` | A `Promise` which resolves to the current [`SignIn`][signin-ref]. |
+
```

Comment:
you didn't know this, so it's okay, but we want to remove the "#### Returns" sections from the docs. I'm thinking we can rephrase like this. This way, the user can find the resource's reference (the link to the signin object) if they want it.

````suggestion
Starts a sign-in flow that uses the OKX Wallet to authenticate the user using their Web3 wallet address. Returns `Promise` which resolves to the current [`SignIn`][signin-ref].

```typescript
function authenticateWithOKXWallet(): Promise<SignInResource>
```
````